lib/eldap/src/eldap.erl

```diff
@@ -483,6 +561,14 @@ parent, `NewSupDN` shall be "".
 ```
 """.
 -doc(#{since => <<"OTP R15B01">>}).
+-spec modify_dn(Handle, Dn, NewRDN, DeleteOldRDN, NewSupDN) -> ok | {ok, Value} | {error, Reason} when
+    Handle :: handle(),
+    Dn :: string(),
+    NewRDN :: string(),
+    DeleteOldRDN :: boolean(),
+    NewSupDN :: string(),
+    Value :: term(),
```

Comment:
Reading the code it looks like this function can also return "[{ok, {referral, referrals()}}](url)"

Reply:
Let me fix it